assesspy/ci.py

```diff
@@ -122,6 +123,6 @@ def prb_ci(
         :func:`boot_ci`
     """
     prb_model = _calculate_prb(estimate, sale_price)
-    prb_ci = prb_model.conf_int(alpha=alpha)[0].tolist()
+    prb_ci = prb_model.conf_int(alpha=alpha)[1].tolist()
```

Comment:
This index also needs to change since 0 now specifies the intercept.

Reply:
**[Question, non-blocking]** I'm a little bit confused by this change. I get why we need to adjust the index into `prb_model.params` in `metrics.prb()`, since it makes sense for `params` to include the intercept, but why is the intercept the first element of [the return value for `conf_int()`](https://www.statsmodels.org/dev/generated/statsmodels.regression.linear_model.RegressionResults.conf_int.html#statsmodels.regression.linear_model.RegressionResults.conf_int)?

Reply:
You can have a confidence interval on $b_0$ as well as $b_1$; it makes perfect sense that `statsmodels` would return it.